python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -650,12 +650,33 @@ def is_valid(self):
         """
         This function checks whether AvgPool2D has compatible attributes with the NPU
         """
+
+        def check_extra_strides():
+            """
+            This function checks whether AvgPool2D could be legalized as ethosu_pooling
+            supported by the NPU.
+            We consider only specific case: when there is no AvgPool2D padding, the spatial
+            dimensions of ifm and the shape of pooling are equal, but stride size exceed 3
+            by any of dimensions, e.g:
+            ifm: (1, 8, 8, _), strides: (8, 8), pool_shape: (8, 8)
+            ifm: (1, 25, 5, _), strides: (25, 5), pool_shape: (25, 5)
+            """
+            if self.pooling_type != "AVG":
```

Comment:
Why can't we use the same trick for max pool as well?

Reply:
Hi @ekalda, exactly, this approach can also be applied to MAX_POOL, done